openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml

```diff
@@ -88,32 +88,128 @@
 - control_key: AU-3
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |
-        A control response is planned. Engineering progress can be tracked via:
+        When the OpenShift API audit system is enabled and configured it generates
+        audit records containing information that establishes the type of event that occurred,
+        when the event occurred, where the event occurred, the source of the event,
+        the outcome of the event, and the idenity of the individuales or subjects associated
+        with the event.  Note,  usage of - oc debug - does not create audit events.
```

Comment:
Can an actual audit record be added to this?

Reply:
To clarify, you're asking if audit records can be added to 'oc debug'?

Reply:
@siliconjesus actaully no. Looking for an actual OpenShift Kubernetes API audit record. For example, a RHEL auditd record looks like:

```
type=SYSCALL msg=audit(1364481363.243:24287): arch=c000003e syscall=2 success=no exit=-13 a0=7fffd19c5592 a1=0 a2=7fffd19c4b50 a3=a items=1 ppid=2686 pid=3538 auid=1000 uid=1000 gid=1000 euid=1000 suid=1000 fsuid=1000 egid=1000 sgid=1000 fsgid=1000 tty=pts0 ses=1 comm="cat" exe="/bin/cat" subj=unconfined_u:unconfined_r:unconfined_t:s0-s0:c0.c1023 key="sshd_config"
type=CWD msg=audit(1364481363.243:24287):  cwd="/home/shadowman"
type=PATH msg=audit(1364481363.243:24287): item=0 name="/etc/ssh/sshd_config" inode=409248 dev=fd:00 mode=0100600 ouid=0 ogid=0 rdev=00:00 obj=system_u:object_r:etc_t:s0  objtype=NORMAL cap_fp=none cap_fi=none cap_fe=0 cap_fver=0
type=PROCTITLE msg=audit(1364481363.243:24287) : proctitle=636174002F6574632F7373682F737368645F636F6E666967
```

What does the equivalent look like for the OpenShift API (not RHCOS)?